tests/chaos/oadp/conftest.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+import pytest
+
+from utilities.oadp import FILE_NAME_FOR_BACKUP, TEXT_TO_TEST, create_rhel_vm, is_storage_class_support_volume_mode
+from utilities.storage import write_file
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def vm_with_datavolume_template(
+    request,
+    chaos_namespace,
+    snapshot_storage_class_name_scope_module,
+):
+    volume_mode = request.param.get("volume_mode")
+    if not is_storage_class_support_volume_mode(
+        storage_class_name=snapshot_storage_class_name_scope_module,
+        requested_volume_mode=volume_mode,
+    ):
+        pytest.skip(
```

Comment:
we should not use programatic skips; you should add proper markers to reflect the needed infra for the test